README.md

```diff
@@ -121,16 +131,21 @@ $ cd ..
 $ cabal repl hasktorch
 ```
 
-We also support Stack with Nix, see below.
+### Troubleshooting Nix-shell under Windows
 
-Note that this shell is configured to use the CPU backend only.
-In order to benefit from any CUDA-capable hardware acceleration your computer may provide (sorry macOS users),
-launch the Nix shell instead with:
+Hasktorch mainly works on Windows Subsystem for Linux 2. 
+Tested on Debian and Ubuntu.
+It is not quite as roubust, but these extra command gets it to work:
 
 ```sh
-$ nix-shell --arg cudaSupport true --argstr cudaMajorVersion 10
+$ nix-shell
+$ cabal update
+$ haskell-language-server
```

Comment:
is this starting the server as a background process?
will vscode be able to connect to it?
I'm asking because I'm used to vscode (specifically, the Haskell extension) starting the haskell language server.

Reply:
No this will run a compile, but it takes a long time and you can find and fix problem this way. Otherwise you are opening VS Code to a sea of red underlines.